### PR TITLE
task: Mount platform-ca secret to pem directory

### PR DESCRIPTION
**Pull request by lee-baines**
Create new object to read ca.pem from platform-ca secret.
Mount platform-ca secret to DS pem directory.
Include new ds-idrepo image with PEM trust manager provider in ds.yaml.

Need to coordinate with forgeops CLOUD-2920-add-pem-image-to-directoryservice-manifests
ref: CLOUD-2920

### CODE REVIEW

**Review by maxres-fr — APPROVED** [review]
LGTM

File: hack/ds.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/name: ds
     app.kubernetes.io/part-of: forgerock
 spec:
-  image: gcr.io/forgeops-public/ds-idrepo:7.1-dev
+  image: gcr.io/forgeops-public/ds-idrepo:7.1-pem
```
Comment (maxres-fr) [inline hack/ds.yaml:10]: you mean to commit this?

Comment (wstrange) [reply hack/ds.yaml:10]: We will need to coordinate the switch with PIT

**Review by jrcast — APPROVED** [review]
LGTM

File: hack/ds.yaml (hunk 1 above)
Comment (lee-baines) [reply hack/ds.yaml:10]: Just trying to work out what to do with making this change in forgeops.  Then I can keep the image the same

Comment (lee-baines) [reply hack/ds.yaml]: reverted this

**Pull request merged**